BUILDING.md

```diff
@@ -82,6 +82,10 @@ This project is built on [GitHub Actions](https://github.com/arduino/arduino-ide
     git push origin 1.2.3
    ```
 
+## Notes for Windows contributors
+In case the build does not succeded it's possible that you have to install Microsoft Visual C++ (MSVC) compiler toolset. You can download it from the Visual Studio [downloads page](https://visualstudio.microsoft.com/downloads/#other) selecting Build Tools for Visual Studio 20XX. Then you have to install Desktop development with C++.
+
```

Comment:
I'll rephrase slightly. MSVC is a strict requirement, and without it the build cannot succeed.

```suggestion
Windows requires Microsoft Visual C++ (MSVC) compiler toolset to be installed on your development machine. In case it's not already present, it can be downloaded from the Visual Studio [downloads page](https://visualstudio.microsoft.com/downloads/#other) selecting Build Tools for Visual Studio 20XX. 
```

Also, not sure it's clear "Then you have to install Desktop development with C++". Is is a separate applicatoin? is it a bundle that must be checked while installing the MSVC?

Reply:
Thank you for your suggestion. I applied the changes here: [5e1d0d00](https://github.com/arduino/arduino-ide/commit/5e1d0d00c8f30c0299b68b27c6318aa07e858848)